src/index.html

```diff
@@ -5,8 +5,228 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo"
+        class="container__logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav__link is-active">apple</a></li>
+          <li><a href="#samsung" class="nav__link">samsung</a></li>
+          <li><a href="#smartphones" class="nav__link">smartphones</a></li>
+          <li>
+            <a href="#laptops" class="nav__link"
+            data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li><a href="#gadgets" class="nav__link">gadgets</a></li>
+          <li><a href="#tablets" class="nav__link">tablets</a></li>
+          <li><a href="#photo" class="nav__link">photo</a></li>
+          <li><a href="#video" class="nav__link">video</a></li>
+        </ul>
+
+      </nav>
+    </header>
+    <main class="hero">
+      <div class="hero__card card" data-qa="card">
+        <img src="images/imac.jpeg" alt="DaMac"
+        class="card__image">
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__rating">
+          <div class="stars card__stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price-container">
+          <div class="card__scary-word">Price:</div>
+          <div class="card__pricetag">$2,199</div>
+        </div>
+        <a class="card__link" href="#totalyotherplace"
+        data-qa="card-hover">Buy</a>
```

Comment:
format according to [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping)